src/installer-skin.lib.php

```diff
@@ -27,6 +27,11 @@
         include_once ABSPATH . '/wp-admin/includes/class-wp-upgrader.php';
         include_once ABSPATH . '/wp-admin/includes/plugin-install.php';
 
+        if (version_compare(SucuriScan::siteVersion(), '5.4', '<')) {
```

Comment:
The breaking change was added in WP 5.3, so the legacy version needs to be loaded when the WP version is < 5.3.

Reply:
Correct, here's an example from the `version_compare` [documentation](https://www.php.net/manual/en/function.version-compare.php):
```php
if (version_compare(PHP_VERSION, '5.0.0', '<')) {
    echo 'I am still PHP 4, my version: ' . PHP_VERSION . "\n";
}
```

Do you think it's more readable to check like this?
```php
if (version_compare(PHP_VERSION, '7.0.0') >= 0) {
    echo 'I am at least PHP version 7.0.0, my version: ' . PHP_VERSION . "\n";
}
```

Reply:
Both work, but I see the first way is used in other places of this codebase, so it makes sense to use that one for consistency reasons.

Reply:
Sorry @gvre, I misunderstood this comment. I have fixed this with https://github.com/Sucuri/sucuri-wordpress-plugin/pull/117/commits/e37ad82eea920e8555c258cc04ad45f2a94d950a :).